cmd/server/pactasrv/admin.go

```diff
@@ -0,0 +1,159 @@
+package pactasrv
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RMI/pacta/db"
+	"github.com/RMI/pacta/oapierr"
+	api "github.com/RMI/pacta/openapi/pacta"
+	"github.com/RMI/pacta/pacta"
+	"go.uber.org/zap"
+)
+
+// Merges two users together
+// (POST /admin/merge-users)
+func (s *Server) MergeUsers(ctx context.Context, request api.MergeUsersRequestObject) (api.MergeUsersResponseObject, error) {
+	req := request.Body
+	actorUserInfo, err := s.getActorInfoOrFail(ctx)
```

Comment:
nit: The `OrFail` seems implicit in the idea that it has a `(_, error)` return signature, no? How about just `s.getActorInfo(ctx)`

Reply:
Good point this is ambiguous.

The "OrFail" is intended to capture "if anonymous what should we do", because I don't want us to handle a logged out user and a user lookup failure the same. I'll change to "OrErrIfAnon"